docs/azure_arc_jumpstart/azure_arc_k8s/aks_stack_hci/aks_hci_powershell/_index.md

```diff
@@ -0,0 +1,139 @@
+Azure Kubernetes Service on Azure Stack HCI is an implementation of AKS on-premises using hyperconverged infrastructure operating system that is delivered as an Azure service.  
+
+  > **Note: Azure Kubernetes Service is now in preview on Azure Stack HCI and on Windows Server 2019 Datacenter.**
+
+  > **Note: Currently, Azure Arc enabled Kubernetes is in [public preview](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/).**
+
+This guide assumes you already have an existing 2-4 node Azure Stack HCI cluster and will leverage PowerShell to automate the AKS creation and onboarding process of the cluster to Azure Arc. 
+
+## Prerequisites
+
+* Clone the Azure Arc Jumpstart repository
+
+    ```shell
+    git clone https://github.com/microsoft/azure_arc.git
+    ```
+  
+* Create Azure service principal (SP)
+
+    To be able to complete the scenario and its related automation, Azure service principal assigned with the “Contributor” role is required. To create it, login to your Azure account run the below command (this can also be done in [Azure Cloud Shell](https://shell.azure.com/)).
+
+    ```powershell
+    Connect-AzAccount
+    $sp = New-AzADServicePrincipal -DisplayName "<Unique SP Name>" -Role 'Contributor'
+   ```
+
+    For example:
+
+    ```powershell
+    $sp = New-AzADServicePrincipal -DisplayName "<Unique SP Name>" -Role 'Contributor'
+    ```
+
+   This command will return a secure string as shown below: 
+
+    ```poweshell
+    Secret                : System.Security.SecureString
+    ServicePrincipalNames : {XXXXXXXXXXXXXXXXXXXXXXXXXXXX, http://AzureArcK8s}
+    ApplicationId         : XXXXXXXXXXXXXXXXXXXXXXXXXXXX
+    ObjectType            : ServicePrincipal
+    DisplayName           : AzureArcK8s
+    Id                    : XXXXXXXXXXXXXXXXXXXXXXXXXXXX
+    Type                  :
+    ```
+
+    To expose the generated password use this code to export the secret: 
+
+    ```poweshell
+    $BSTR = [System.Runtime.InteropServices.Marshal]::SecureStringToBSTR($sp.Secret)
+    $UnsecureSecret = [System.Runtime.InteropServices.Marshal]::PtrToStringAuto($BSTR)
+    ```
+
+    > **Note: It is optional but highly recommended to scope the SP to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/powershell/module/az.resources/new-azadserviceprincipal?view=azps-5.4.0)**
+
+* Enable your subscription with the two resource providers for Azure Arc enabled Kubernetes. Registration is an asynchronous process, and registration may take approximately 10 minutes.
+
+  ```powershell
+  Register-AzResourceProvider -ProviderNamespace Microsoft.Kubernetes
+  Register-AzResourceProvider -ProviderNamespace Microsoft.KubernetesConfiguration
+  ```
+* Since AKS on Azure Stack HCI is in public preview you will need to download the required software for evaluation from the [registration page](https://aka.ms/AKS-HCI-Evaluate). Complete the preview registration form and safe the zip file to the "Downloads" folder.
```

Comment:
```suggestion
* Since AKS on Azure Stack HCI is in preview you will need to download the required software for evaluation from the [registration page](https://aka.ms/AKS-HCI-Evaluate). Complete the preview registration form and save the zip file to the "Downloads" folder.
```